.github/workflows/build.yaml

```diff
@@ -89,9 +89,9 @@ jobs:
         run: bash ./gradlew buildPlugin -PuseLocal=${{ env.USE_LOCAL_PLUGIN }}
       - name: 'Archive artifacts'
         if: ${{ runner.os == 'Linux' && !failure() }}
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4.3.4
         with:
-          name: liberty-tools-intellij-${{ github.sha }}
+          name: liberty-tools-intellij-${{ github.sha }}-${{ env.REF_LSP4IJ }}
```

Comment:
@anusreelakshmi934 , I am hoping that this artifact name will repeat once this PR (https://github.com/OpenLiberty/liberty-tools-intellij/pull/864) is merged. Make sure to adjust the artifact name to something like this
`liberty-tools-intellij-${{ github.sha }}-${{ env.REF_LSP4IJ }}-${{ env.REF_LTI_TAG }}`

Reply:
I believe that change can be done in the PR for Run LSP4iJ continuous integration builds with existing LTI releases . Because this PR doesn't contain the changes to run against existing LTI releases

Reply:
Yes true @anusreelakshmi934